### ISSUE TITLE
include and exclude have no effect

### ISSUE THREAD

**Issue by GreenDjango** [tsconfig/bases#207]
Hi,

`tsconfig.json` files specified using `extends` will have `include` and `exclude` paths relative to their location, as per the [include docs](https://www.typescriptlang.org/tsconfig#include):

> These filenames are resolved relative to the directory containing the tsconfig.json file.

So a config like [Next.js base](https://github.com/tsconfig/bases/blob/main/bases/next.json) install with `npm` will produce a typescript error:

> error TS18003: No inputs were found in config file '/home/user/my-project/tsconfig.json'.
> Specified 'include' paths were '["node_modules/@tsconfig/next/next-env.d.ts","node_modules/@tsconfig/next/**/*.ts",...]'
> and 'exclude' paths were '["node_modules/@tsconfig/next/node_modules"]'.

for a tsconfig.json like:

```json
{
  "extends": ["@tsconfig/next/tsconfig.json"],
}
```

Not specify `include` and `exclude` in base template can remove ambiguity

*Have a nice day*

**Comment by zetaraku** [comment]
Agreed. `include` and `exclude` in the `@tsconfig/next/tsconfig.json` file are useless and misleading.

Maybe we can remove that and provide this excerpt in the README file and instruct users to manually add it:

```json
{
  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
  "exclude": ["node_modules"]
}
```

**Comment by joshunger** [comment]
Wouldn't using `${configDir}` in `include` work in base?

**Comment by vladshcherbin** [comment]
The main readme file is misleading and the example code it has is not working due to this behavior.

**Comment by orta** [comment]
You are welcome to add notices to the configs when it makes sense, like we did with `baseUrl` in https://github.com/tsconfig/bases/pull/248

**Comment by vladshcherbin** [comment]
@orta the whole `next` config file is outdated and gives errors even by `next cli` itself

I can update the docs and the configuration to [latest version](https://github.com/vercel/next.js/blob/v15.1.0/packages/create-next-app/templates/app/ts/tsconfig.json) which `next` team actually uses and provides in new projects since next 15 if you accept a major version bump

I also found previous attempts of similar changes in https://github.com/tsconfig/bases/pull/232 and https://github.com/tsconfig/bases/pull/243 but none of them landed. Just asking before making changes if you are okay with them

**Comment by orta** [comment]
Strange, I accepted one, unsure why it didn't land. Seems fine with me
